deployment/I_exrm_releases.md

```diff
@@ -297,10 +355,11 @@ Now let's try connecting a console to the running release with the `remote_conso
 
 ```console
 $ rel/hello_phoenix/bin/hello_phoenix remote_console
-Erlang/OTP 17 [erts-6.0] [source-07b8f44] [64-bit] [smp:8:8] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]
+Using /home/sashaafm/Documents/hello_phoenix/rel/hello_phoenix/releases/0.0.1/hello_phoenix.sh
+Erlang/OTP 18 [erts-7.2] [source] [64-bit] [smp:2:2] [async-threads:10] [hipe] [kernel-poll:false]
 
-Interactive Elixir (1.0.2) - press Ctrl+C to exit (type h() ENTER for help)
-iex(hello_phoenix@127.0.0.1)1>
+Interactive Elixir (1.2.0) - press Ctrl+C to exit (type h() ENTER for help)
+iex(hello_phoenix@LXLE)1> 
```

Comment:
Most people I believe will have 127.0.0.1 if running this locally. May want to swap this out.